### DISCUSSION TITLE
Is Flet's Audio plugin going to get more features such as background audio support?

### DISCUSSION THREAD

**Discussion by litmount** [flet-dev/flet#3291] [General]
Hello,

It is really nice that Flet has and audio plugin but for me it is useless for my project. It seems that there is no way to tie into Android or IOS for background audio/screen off. I was trying to make a custom audio player for our company's podcast and had it fully functional when realize I wont be able to use it. 

Flet's audio plugin needs to integrate something like this: https://github.com/ryanheise/audio_service/tree/minor/audio_service

I tried to see about integrating myself from the documentation but it all says "coming soon"

I really think Flet is great and I really appreciate the project! However I do think this is one of the two big things needed for it to move forward for mobile apps. The other would be cross platform integration of notifications. I can't seem to find any concrete on where that is either.

Basically I'm  just checking to see if any of this is on the roadmap or if there are any work arounds. Thanks!

**Comment by ndonkoHenri** [comment]
Nice suggestion.
Will look into it...

**Reply by litmount** [reply]
@ndonkoHenri Did you get a change to look into this then?

**Reply by syleishere** [reply]
Better off using flet_video with audio_service as video at least has playlist support where audio doesn't. Love to see audio_service wrapping video

**Reply by ndonkoHenri** [reply]
> Did you get a change to look into this then?

Not yet :)

**Reply by litmount** [reply]
Just wanted to check in One more time. Is something like this on the road map at all?
